src/lib/simulation/toa.ts

```diff
@@ -953,7 +958,12 @@ async function calcTOAInput({
 }> {
 	const cost = new Bank();
 	const kc = kcOverride ?? (await getMinigameScore(user.id, 'tombs_of_amascut'));
-	cost.add('Super combat potion(4)', quantity);
+	if (minimumSuppliesNeeded.has('Super combat potion(4)')) {
+		cost.add('Super combat potion(4)');
+	} else if (minimumSuppliesNeeded.has('Super attack(4)') && minimumSuppliesNeeded.has('Super strength(4)')) {
```

Comment:
Doesn't necessarily require an else if, I think an else would suffice here. 
If neither are present what would be the cost for example?

Reply:
My thoughts here were the same. Pretty much checking if either are available before adding to the cost in order to avoid null errors but it should return higher up before if gets here if neither is available anyways.